### ISSUE TITLE
universal accesory setprobs

### ISSUE THREAD

**Issue by habeck76** [HM-RedMatic/RedMatic-HomeKit#111]
Kurze Frage: Ist es Möglich die maxValue der TargetTemperature eines Thermostats oder HeaterCoolers im Flow zu erhöhen. Ich würde gerne die Soll- und Istwerte meiner Heizkreisläufe darüber abbilden. @hobbyquaker: Du hattest in #44 mal etwas in der Richtung erwähnt. Danke!

**Comment by hobbyquaker** [comment]
Ja, einfach eine Nachricht mit den Properties an die entsprechende Characteristik senden. Beispiel:

![image](https://user-images.githubusercontent.com/2503307/51426509-319b3d80-1bec-11e9-82d1-13872dd9c77c.png)

Also eine Nachricht in der Form
```
{
  "topic": "0/TargetTemperature",
  "payload": {"minValue":10, "maxValue":80}
}
```

**Comment by drsnuggles1982** [comment]
Hey,

Muss man noch was beachten? Hab mir die payload für die CurrentTemperature gesetzt mit minValue: -20.0
minStep: 0.1

Leider passiert gar nichts auch nach einem kompletten Neustart des Systems.

Gruß
Und Danke für Hinweise 
Felix
